routes/web.php

```diff
@@ -21,12 +20,16 @@
     return view('welcome');
 });
 
-Route::get('/login', [LoginController::class, 'login'])->name('auth.loginForm');
+Route::group(['prefix' => 'cms'], function () {
+    Route::get('/login', [LoginController::class, 'login'])->name('auth.loginForm');
 
-Route::post('/login', [LoginController::class, 'handelLogin'])->name('auth.login');
+    Route::post('/login', [LoginController::class, 'handelLogin'])->name('auth.login');
 
-Route::get('/logOut', [LoginController::class, 'logOut'])->name('auth.logout');
+    Route::get('/logOut', [LoginController::class, 'logOut'])->name('auth.logout');
 
-Route::get('/register', [RegisterController::class, 'register'])->name('auth.registerForm');
+    Route::get('/register', [RegisterController::class, 'register'])->name('auth.registerForm');
 
-Route::post('/register', [RegisterController::class, 'handelRegister'])->name('auth.register');
+    Route::post('/register', [RegisterController::class, 'handelRegister'])->name('auth.register');
+});
+
+Route::resource('partner', PartnerController::class);
```

Comment:
```suggestion
Route::resource('partners', PartnerController::class);
```
sửa nốt cái này cho đúng chuẩn này